packages/eas-cli/src/credentials/ios/IosCredentialsProvider.ts

```diff
@@ -1,22 +1,19 @@
 import { Platform } from '@expo/eas-build-job';
 import { CredentialsSource, IosDistributionType, IosEnterpriseProvisioning } from '@expo/eas-json';
+import nullthrows from 'nullthrows';
 
-import { CredentialsManager } from '../CredentialsManager';
 import { Context } from '../context';
 import * as credentialsJsonReader from '../credentialsJson/read';
-import type { IosCredentials } from '../credentialsJson/read';
-import { SetupBuildCredentials } from './actions/SetupBuildCredentials';
-import { AppLookupParams } from './api/GraphqlClient';
+import { SetupMultitargetBuildCredentials } from './actions/SetupMultitargetBuildCredentials';
 import { IosCapabilitiesOptions } from './appstore/ensureAppExists';
+import { App, IosCredentials, Target } from './types';
 import { isAdHocProfile } from './utils/provisioningProfile';
 
-export type { IosCredentials };
-
 interface Options {
-  app: AppLookupParams;
+  app: App;
+  targets: Target[];
   distribution: IosDistributionType;
   enterpriseProvisioning?: IosEnterpriseProvisioning;
-  skipCredentialsCheck?: boolean;
```

Comment:
Why are you removing this option?

Reply:
Discussed offline - the flag is ignored since https://github.com/expo/eas-cli/pull/344